src/PhpBrew/Build.php

```diff
@@ -83,6 +90,35 @@ public function __construct($version, $name = null, $installPrefix = null)
         $this->osRelease = php_uname('r');
     }
 
+    public function getPkgConfigPaths()
+    {
+        return array_keys($this->pkgConfigPaths);
+    }
+
+    public function addPkgConfigPath($path)
+    {
+        $this->pkgConfigPaths[$path] = true;
+    }
+
+    public function putPkgConfigPathsEnv()
+    {
+        $paths = $this->getPkgConfigPaths();
+        $pathCount = count($paths);
+
+        if ($pathCount == 0) {
+            return false;
```

Comment:
This return value isn't used anywhere, so the condition itself isn't relevant as well. Please remove the `if`/`else` and return and leave only the code under `else`.